### PR TITLE
OEUI-58: Implement tab functionality for standard dosing and free text(code optimization)

### PR DESCRIPTION
**Pull request by geofrocker**
## JIRA TICKET NAME:
[OEUI-58:Implement tab functionality for standard dosing and free text](https://issues.openmrs.org/browse/OEUI-58)
### SUMMARY:
This entails enabling a user to either choose standard dosing or free text format of adding instructions when adding a new drug order or editing active drug order. This also includes updating the store on which format the user is using.

### CODE REVIEW

**Comment by geofrocker** [comment]
Thanks @kwahalf. I am going to work on it

**Comment by geofrocker** [comment]
@kwahalf Fixed

File: app/js/components/tabs/DosageTabs.jsx [hunk 1]
```diff
@@ -3,58 +3,46 @@ import PropTypes from 'prop-types';
 import DosageTab from '../tabs/DosageTab';
 
 class DosageTabs extends Component {
-  constructor(props) {
-    super(props);
+  constructor(props, context) {
```
Comment (kwahalf) [inline app/js/components/tabs/DosageTabs.jsx]: are you using context anywhere?

Comment (geofrocker) [reply app/js/components/tabs/DosageTabs.jsx]: Fixed. thank u

**Pull request merged**